lang/java/reef-io/src/main/java/org/apache/reef/io/network/impl/NetworkConnectionServiceImpl.java

```diff
@@ -148,7 +161,7 @@ public void onNext(final Identifier id) {
       throw new NetworkException("ConnectionFactory " + connFactoryId + " was already registered.");
     }
     final ConnectionFactory connFactory = connFactoryMap.putIfAbsent(id,
-        new NetworkConnectionFactory<>(this, id, codec, eventHandler, linkListener));
+        new NetworkConnectionFactory<>(this, connFactoryId, codec, eventHandler, linkListener, null));
```

Comment:
If the suggestion in the previous comment is taken, this would call the deprecated constructor.